README.md

```diff
@@ -29,7 +29,12 @@ docker-compose up
 
 The app should be live at [http://localhost:8080](http://localhost:8080)
 
+## Self-Hosting
+
+As we believe in decentralization at all layers, we've made it easy to host your own Balancer Frontend.
+
```

Comment:
Here we should probably add a small section about dependencies and provide links to the apps they need to have accounts for. e.g. Infura, Alchemy and blocknative.